packages/affiliates/scripts/example-transactions.js

```diff
@@ -0,0 +1,37 @@
+const { BigQuery } = require("@google-cloud/bigquery");
+const highland = require("highland");
+const Queries = require("../libs/bigquery");
+const moment = require("moment");
+
+const contract = "0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56";
+const start = moment("2020-10-06", "YYYY-MM-DD").valueOf();
+const end = moment("2020-10-10", "YYYY-MM-DD").valueOf();
+
+const client = new BigQuery();
+const queries = Queries({ client });
+
+async function runTest() {
+  // returns a node read stream
+  const stream = await queries.streamTransactionsByContract(contract, start, end);
```

Comment:
So which sort of transactions do you intend to track? Am i reading it correctly that you are looking at ALL transactions involving the contract? (i.e. the `toAddress` is the contract)